docs/source/conf.py

```diff
@@ -18,23 +18,62 @@
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-extensions = ['sphinx.ext.autodoc','numpydoc',
-    'sphinx.ext.todo','sphinx.ext.viewcode']
+extensions = ['sphinx.ext.autodoc', 'numpydoc',
+              'sphinx.ext.todo', 'sphinx.ext.viewcode', 'nbsphinx']
 
 templates_path = ['_templates']
 exclude_patterns = []
 
 
-
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = 'sphinx_rtd_theme'
+html_theme = "sphinx_book_theme"
 html_static_path = ['_static']
+
 numpydoc_show_class_members = False
 numpydoc_class_members_toctree = False
-numpydoc_show_inherited_class_membersbool=False
+numpydoc_show_inherited_class_membersbool = False
 
 autodoc_default_options = {
     'special-members': '__init__'
-}
\ No newline at end of file
+}
+
+html_title = "Ablator Documentation"
+html_favicon = '_static/ablator-logo.svg'
+
+# html_theme_options = {
```

Comment:
Are the comments required?

Reply:
These are not required. I have removed them.